redbot/cogs/customcom/customcom.py

```diff
@@ -206,6 +206,39 @@ async def customcom(self, ctx: commands.Context):
         """Custom commands management."""
         pass
 
+    @customcom.command(name="raw")
+    async def cc_raw(self, ctx: commands.Context, command: str.lower):
+        """Get the raw response of a custom command, to get the proper markdown.
+        
+        This is helpful for copy and pasting."""
+        commands = await self.config.guild(ctx.guild).commands()
+        if command not in commands:
+            return await ctx.send("That command doesn't exist.")
+        command = commands[command]
+        if isinstance(command["response"], str):
+            raw = discord.utils.escape_markdown(command["response"])
+            await ctx.send(raw)
+        else:
+            msglist = []
+            if await ctx.embed_requested():
+                colour = await ctx.embed_colour()
+                for number, response in enumerate(command["response"], start=1):
+                    raw = discord.utils.escape_markdown(response)
+                    embed = discord.Embed(
+                        title=_("Response #{num}/{total}".format(num=number, total=len(command["response"]))),
```

Comment:
String needs to be formatted *after* translation.
```suggestion
                        title=_("Response #{num}/{total}").format(num=number, total=len(command["response"])),
```